.rubocop.yml

```diff
@@ -2,7 +2,7 @@ inherit_from: .rubocop_todo.yml
 
 AllCops:
   DisplayCopNames: true
-  TargetRubyVersion: 2.0
+  TargetRubyVersion: 2.1
```

Comment:
OOC why did this have to change?

Reply:
It doesn't, but we already dropped support for 2.0 in version 4.0.